### PR TITLE
Added a simple cdn for images or in general files for example videos too

### PR DESCRIPTION
**Pull request by DoggoITA**

### CODE REVIEW

File: webcdn.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'shopdeck.settings')
+from django.core.wsgi import get_wsgi_application
+
+application = get_wsgi_application()
+from shopdeckdb.models import *
+from django.core.exceptions import ObjectDoesNotExist
+
+webcdn_directory = 'webcdn'
+
+print("CDN (for images, etc.) Services Starting Up")
```
Comment (Rix565) [inline webcdn.py:13]:
```suggestion
print("Local CDN Starting Up")
```

File: main.py [hunk 2]
```diff
@@ -6,12 +6,13 @@
 print("----------------------------------")
 
 from flask import Flask
-import ecs, ias, cas, cdn
+import ecs, ias, cas, cdn, webcdn
```
Comment (Rix565) [inline main.py:9]: rename to assetcdn?

File: main.py [hunk 3]
```diff
@@ -6,12 +6,13 @@
...
 
 app = Flask(__name__)
 app.register_blueprint(ecs.ecs)
 app.register_blueprint(ias.ias)
 app.register_blueprint(cas.cas)
 app.register_blueprint(cdn.ccs)
+app.register_blueprint(webcdn.cdn)
```
Comment (Rix565) [inline main.py:16]: (refer to previous comment)

**Review by Rix565 — CHANGES_REQUESTED** [review]
neat!

**Comment by DoggoITA** [comment]
done

**Comment by Rix565** [comment]
neat!!! merged

**Pull request merged**